extensions/amp-story/1.0/localization.js

```diff
@@ -95,6 +94,9 @@ const LANGUAGE_CODE_CHUNK_REGEX = /\w+/gi;
  * @return {!Array<string>} A list of language codes.
  */
 export function getLanguageCodesFromString(languageCode) {
+  if (!languageCode || languageCode === '') {
```

Comment:
This works, but actually so does just

```javascript
if (!languageCode) {
  ...
}
```

Empty string is considered "falsy" in JavaScript, so `!""` evaluates to true

Reply:
Ahh I see! Thanks for the help!